packages/angular/projects/ui-angular/src/lib/utilities/amplify-slot/amplify-slot.component.ts

```diff
@@ -13,17 +13,17 @@ import { CustomComponentsService } from '../../services/custom-components.servic
 })
 export class AmplifySlotComponent implements AfterContentInit {
   @Input() name: string;
-  @Input() context: Record<PropertyKey, any>;
+  @Input() context: Record<PropertyKey, unknown>;
```

Comment:
This is the weird one. We know `context` is some keyed `object`, but we cannot know what the type is specifically because this is a generic slot component.

Reply:
Angular [uses](https://github.com/angular/angular/blob/14.3.0/packages/common/src/directives/ng_template_outlet.ts#L48) `Object`, but that's also not recommended for good reasons. I'm good to leave this as `Record<PropertyKey, unknown>`, or maybe `unknown`. Any opinions are welcome.